src/components/input/TextComponent.tsx

```diff
@@ -23,21 +25,26 @@ type TextComponentProps = {
   /** React Element to display inside the Field, on the right side */
   rightDecorators?: JSX.Element | JSX.Element[];
   className?: string;
+  /** When present, it specifies that the field is disabled. */
   disabled?: boolean;
   /** React Element to display inside the Field, on the left side */
   iconElement?: JSX.Element;
   id?: string;
   label?: string;
   /** Force the text to display masked "••••" */
   isMasked?: boolean;
-  /** Deprecated, please use rightDecorators instead */
+  /** @deprecated Deprecated, please use rightDecorators instead */
   masked?: boolean;
   placeholder?: string;
   onClick?: () => any;
   onFocus?: (event?: React.FocusEvent<HTMLElement>) => any;
   onKeyDown?: (event) => any;
-  value?: string;
   showRequired?: boolean;
+  /** When present, it specifies that the field is read-only. */
+  readOnly?: boolean;
+  /** Size of the input */
+  size?: 'small' | 'medium';
```

Comment:
i though there were large input also

Reply:
large for Select ! maybe large input in a next iteration